Libplanet/Tx/TxExecution.cs

```diff
@@ -42,11 +52,18 @@ private protected TxExecution(BlockHash blockHash, TxId txId)
         [Pure]
         public TxId TxId { get; }
 
+        /// <summary>
+        /// The logs recorded while executing <see cref="Transaction{T}"/>'s actions.
+        /// </summary>
+        [Pure]
+        public List<List<string>>? ActionsLogsList { get; }
```

Comment:
I'm curious what the rationale is for having the ActionLogList field to be nullable.

Reply:
I thought `TxExecution.ActionsLogsList` should be nullable because the previous `TxExecution`s don't have the `ActionsLogsList` yet.

Reply:
Makes sense 😅